docs/docs/rest-api/public/api/v2/types/pod.raml

```diff
@@ -171,3 +171,9 @@ types:
           The role to use. If not specified, uses the role of the enclosing group, or the default role.
           At the moment, only the default role or the group role are allowed
       linuxInfo?: linux.LinuxInfo
+      legacySharedResources?:
```

Comment:
"shared resources" sounds more general than shared cgroup. for example, "resources" like persistent volumes, root file system, sandbox are not shared even in the legacy world. how about "legacySharedCgroups"?

Reply:
That sounds fine to me